pages/advanced/api-new-networks.md

```diff
@@ -0,0 +1,49 @@
+## New Supported Chains
+
+Safe's vision is to make every Web3 account a smart account. Therefore, we prioritize teams and chains building with smart accounts, especially if they push account abstraction (for example, ERC-4337/EIP-1271 adoption) with Safe (learn more [here](https://docs.safe.global/home/4337-safe)).
+
+To have the full Safe stack running on your chain network. You need to consider each of these steps:
+
+### Deploy Safe canonical contracts
+
+**Process**
+
+- Follow all the steps in the [safe-contracts](https://github.com/safe-global/safe-contracts#deploy) GitHub repository.
+- Create a Pull Request in the [safe-deployments](https://github.com/safe-global/safe-deployments) GitHub repository. 
+
+**Cost**
+
+The contracts are fully open source, but gas is needed in the native currency of the new chain for contract deployment.
+
+**Timeline**
+
+Pull Requests on Safe's GitHub are reviewed roughly on a two-week cadence.
+
+**EVM only**
+
+Safe's security is based on the contracts deployed in EVM-compatible networks. For the time being, we are not looking to extend beyond EVM.
+
+### Safe{Wallet} interface and Safe{Core} infrastructure
+
+**Process**
```

Comment:
I find this section a bit confusing, can we reword and restructure this? Happy to chat sync to explain what I find confusing.